dnf-docker-test/features/module-install-pkg-3.feature

```diff
@@ -27,3 +28,26 @@ Feature: Installing package when default stream is defined
           | version | -1    |
           | enabled | 1     |
           | stream  | f26   |
+
+  Scenario: cleanup from previous scenario
+      Given I successfully run "dnf remove TestY -y"
+        And I successfully run "dnf module disable ModuleY"
+
+  Scenario: rpm from enabled stream is preferred regardless of NVRs
+      Given I successfully run "dnf module enable ModuleY:f26"
+       When I save rpmdb
+        And I successfully run "dnf install -y TestY"
+       Then rpmdb changes are
+          | State     | Packages       |
+          | installed | TestY/1-1.modY |
+
+  Scenario: cleanup from previous scenario
+      Given I successfully run "dnf remove TestY -y"
+        And I successfully run "dnf module disable ModuleY"
+
+  Scenario: rpm from default stream is preferred regardless of NVRs
```

Comment:
I believe this is the same as you first scenario
Scenario: I can install a specific package from a module not enabled when default stream is defined

Reply:
You are right. I'll remove it.